### PR TITLE
fix(rust-job): copy-edit, active voice AND feat(rust-job): add APPLICATION field

### PR DESCRIPTION
**Pull request by yuuyins**
### Description of changes

Initially, just some few copy edits including active voice, and adding an application/contact field which I see many posts on https://reddit.com/r/rust/comments/13yx1dn do.

### Additional comments

In the future, I want to turn this into a more precise template for those looking for jobs following an example from a company using Rust I saw before (if I remember correctly, they call it performance application or similar that as to state clearly what the candidate must do on the jobs, what will likely use like technologies/architectures, requirements, etc, instead of solely focusing on experience level / seniority).

### CC
@nohsheef

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rust-job.md [hunk 1]
```diff
@@ -7,31 +7,32 @@ assignees: ''
...
-__DESCRIPTION__: [What does your company do, and what are you using Rust for? How much experience are you seeking and what seniority levels are you hiring for? The more details the better.]
+__DESCRIPTION__: [What does your organization do, and what are you using Rust for? How much experience are you seeking and what seniority levels are you hiring for? The more relevant details for the job, the better.]
 
-__ESTIMATED COMPENSATION__: [Be courteous to your potential future colleagues by attempting to provide at least a rough expectation of wages/salary.
+__ESTIMATED COMPENSATION__: [Provide at least a rough expectation of wages/salary, as a courteous behavior to your potential future colleagues.]
 
-If you are listing several positions in the "Description" field above, then feel free to include this information inline above, and put "See above" in this field.
+__APPLICATION__: [Forms, or contacts like e-mails for the candidate to apply for.]
+
+If you are listing multiple positions in the "Description" field above, then feel free to include this information inline above, and put "See above" in this field.
```
Comment (nohsheef) [inline .github/ISSUE_TEMPLATE/rust-job.md:26]: As an applicant, I would actually prefer e-mail as 1st contact. Maybe we can word it as preferred medium?

File: .github/ISSUE_TEMPLATE/rust-job.md [hunk 2]
```diff
@@ -7,31 +7,32 @@ assignees: ''
...
 
 If compensation is negotiable, please attempt to provide at least a base estimate from which to begin negotiations. If compensation is highly variable, then feel free to provide a range.
 
-If compensation is expected to be offset by other benefits, then please include that information here as well. If you don't have firm numbers but do have relative expectations of candidate expertise (e.g. entry-level, senior), then you may include that here.
-If you truly have no information, then put "Uncertain" here.
+If you expect compensation to be offset by other benefits, then include that information as well. If you do not have firm numbers but do have relative expectations of candidate expertise (e.g. entry-level, senior), then you may include that. If you truly have no information, then put "Uncertain" here.
```
Comment (nohsheef) [inline .github/ISSUE_TEMPLATE/rust-job.md:30]: Missing comma before ` but do have`, which was missing in original version?

File: .github/ISSUE_TEMPLATE/rust-job.md [hunk 3]
```diff
@@ -7,31 +7,32 @@ assignees: ''
...
 
-Note that some jurisdictions (e.g., California, Colorado, New York City) currently or will soon require salary ranges on job postings by law. If your company is based in one of these locations or you plan to hire employees who reside in any of these locations, you are likely subject to these laws. Other jurisdictions may require salary information to be available upon request or be provided after the first interview. To avoid issues, we recommend all postings provide salary information.
+Note that some jurisdictions (e.g. United States' jurisdictions of California, Colorado, and New York City) currently or will legally require salary ranges on job postings. If your organization is in one of these locations or you plan to hire employees who reside in any of these locations, you are likely subject to these laws. Other jurisdictions may require salary information to be available upon request or that you provide it after the first interview. To avoid issues, we recommend all postings to provide salary information.
```
Comment (nohsheef) [inline .github/ISSUE_TEMPLATE/rust-job.md:32]: Is the part about _currently or soon_ still relevant? Can we simplify it? If not, the new version sounds like it's missing a verb after _currently_, don't you think?

File: .github/ISSUE_TEMPLATE/rust-job.md [hunk 4]
```diff
@@ -7,31 +7,32 @@ assignees: ''
...
 
-You must state clearly in your posting if you are planning to compensate employees partially or fully in something other than fiat currency (e.g. cryptocurrency, stock options, equity, etc).
+You must state in your posting if you are planning to compensate employees partially or fully in something other than fiat currency (e.g. cryptocurrency, stock options, equity, etc).
 
 Do not put just "Uncertain" in this case as the default assumption is that the compensation will be 100% fiat.
 
-Postings that fail to comply with this addendum will be removed. Thank you.]
+[rustgig](https://github.com/rustgig) team will remove postings that fail to comply with this addendum. Thank you.
```
Comment (nohsheef) [inline .github/ISSUE_TEMPLATE/rust-job.md:38]: Missing terminating `]`?

**Review by nohsheef — COMMENTED** [review]
Minor nitpicks. I like your changes, so feel free to merge with further changes or after correcting my comments.

File: .github/ISSUE_TEMPLATE/rust-job.md (hunk 4 above)
Comment (yuuyins) [reply .github/ISSUE_TEMPLATE/rust-job.md]: Re-added.

File: .github/ISSUE_TEMPLATE/rust-job.md (hunk 1 above)
Comment (yuuyins) [reply .github/ISSUE_TEMPLATE/rust-job.md:26]: What do you think this way?

> Ideally include e-mail address contact for the candidate to apply for the job, but you may also add other application mediums such as forms.

https://github.com/rustgig/gigs/pull/1/files#diff-bec60ed09f5a4329950cd6861e0239414486b47314d7c20209f919bbc368bdf5R24

File: .github/ISSUE_TEMPLATE/rust-job.md (hunk 3 above)
Comment (yuuyins) [reply .github/ISSUE_TEMPLATE/rust-job.md]: These sources 

> California job postings will soon include pay ranges, thanks to a new salary transparency bill [signed into law](https://news.bloomberglaw.com/daily-labor-report/california-employers-to-disclose-pay-ranges-with-newsom-signing) by Gov. Gavin Newsom Tuesday.
>
> **The law, which goes into effect Jan. 1, 2023**
---[Jennifer Liu](https://www.cnbc.com/jennifer-liu/), https://www.cnbc.com/2022/09/28/california-pay-transparency-law-to-require-salary-ranges-on-job-postings.html

> A Colorado law requires employers to disclose their targeted pay range in job postings.
> ...
> **Colorado's [Equal Pay for Equal Work Act](https://leg.colorado.gov/sites/default/files/2019a_085_signed.pdf) took effect on [2021] Jan. 1.**
>
>---[Lisa Nagele-Piazza](https://www.shrm.org/authors/Pages/Lisa-Nagele-Piazza.aspx), https://www.shrm.org/resourcesandtools/legal-and-compliance/state-and-local-updates/pages/some-employers-are-excluding-colorado-applicants-for-remote-work.aspx

> Ads for jobs in **New York City must include a salary range starting Tuesday [2022 November 1]**, under a new pay transparency law that’s forcing employers to rethink their hiring practices and pay scales.
>
> [New York City Local Law 59](https://aboutblaw.com/5x5)....
>
>---Chris Marr, https://news.bloomberglaw.com/daily-labor-report/pay-transparency-in-new-york-city-new-job-posting-law-explained 

say the laws requiring salary ranges on job postings in the United States' jurisdictions of California, Colorado, and New York City, respectively, are all in effect already.

Thus, "currently or soon" is not relevant anymore. I removed it.

File: .github/ISSUE_TEMPLATE/rust-job.md (hunk 4 above)
Comment (nohsheef) [reply .github/ISSUE_TEMPLATE/rust-job.md]: LGTM

**Review by nohsheef — APPROVED** [review]

**Comment by yuuyins** [comment]
> Minor nitpicks. I like your changes, so feel free to merge with further changes or after correcting my comments.

It seems like I do not have write access to https://github.com/rustgig/gigs, so I cannot merge the pull request.

2023-07-04T22-12-03UTC

More information:
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request
- https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github

**Comment by nohsheef** [comment]
Fixes base permissions for the org, which was r/o by default for members.

**Pull request merged**